notebooks/test_watertank.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import annotations
+
+import threading
+import time
+
+import numpy as np
+
+from openlifu.bf.pulse import Pulse
+from openlifu.bf.sequence import Sequence
+from openlifu.geo import Point
+from openlifu.io.LIFUInterface import LIFUInterface
+from openlifu.plan.solution import Solution
+
+# set PYTHONPATH=%cd%\src;%PYTHONPATH%
+# python notebooks/test_watertank.py
+
+"""
+Test script to automate:
+1. Connect to the device.
+2. Test HVController: Turn HV on/off and check voltage.
+3. Test Device functionality.
+"""
+
+log_interval = 1  # seconds; you can adjust this variable as needed
+stop_logging = False  # flag to signal the logging thread to stop
+
+def log_temperature():
+    # Create a file with the current timestamp in the name
+    timestamp = time.strftime("%Y%m%d_%H%M%S")
+    filename = f"{timestamp}_temp.log"
+    with open(filename, "w") as logfile:
+        while not stop_logging:
+            temperature = interface.txdevice.get_temperature()
+            a_temp = interface.txdevice.get_ambient_temperature()
+            current_time = time.strftime("%Y-%m-%d %H:%M:%S")
+            log_line = f"{current_time}: Temperature: {temperature}, Ambient Temperature: {a_temp}\n"
+            logfile.write(log_line)
+            logfile.flush()  # Ensure the data is written immediately
+            time.sleep(log_interval)
+
+
+print("Starting LIFU Test Script...")
+interface = LIFUInterface(test_mode=False)
+tx_connected, hv_connected = interface.is_device_connected()
+if tx_connected and hv_connected:
+    print("LIFU Device Fully connected.")
+else:
+    print(f'LIFU Device NOT Fully Connected. TX: {tx_connected}, HV: {hv_connected}')
```

Comment:
curious, would this connectivity information also eventually be contained in a call to `LIFUInterface.get_status`?